test/extensions/filters/network/common/fuzz/uber_per_readfilter.tmpl.cc

```diff
@@ -19,20 +19,11 @@ static const int SecondsPerDay = 86400;
 std::vector<absl::string_view> UberFilterFuzzer::filterNames() {
   // Add filters that are in the process of being or are robust against untrusted downstream
   // traffic.
+  static const std::vector<std::string> supported_filter_names = {{{FILTERS}}};
```

Comment:
I think this vector should be the same as the one returned by the `Registry::FactoryRegistry<Server::Configuration::NamedNetworkFilterConfigFactory>::registeredNames()`. Is not the case? It is fully determined by what factories were linked into the binary, so may not need to have this list generated into the source file.

Reply:
Yes, that is the case and works fine.